Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -385,20 +423,48 @@ private Order GetOrderByIdInternal(int orderId)
         public Order GetOrderByBrokerageId(string brokerageId)
         {
             // this function can be invoked by brokerages when getting open orders, guard against null ref
-            if (_orders == null) return null;
+            if (_closedOrders == null || _openOrders == null) return null;
 
-            var order = _orders.FirstOrDefault(x => x.Value.BrokerId.Contains(brokerageId)).Value;
+            var order =
+                _openOrders.FirstOrDefault(x => x.Value.BrokerId.Contains(brokerageId)).Value ??
+                _closedOrders.FirstOrDefault(x => x.Value.BrokerId.Contains(brokerageId)).Value;
```

Comment:
nit - just my personal style preference, but when splitting new lines, I tend to prefer the operator at the beginning of the line as it makes it easier to see (especially when lines get long)

Reply:
your right, it's easier to read. I'll change it!